src/logic/Game/Game.tsx

```diff
@@ -0,0 +1,45 @@
+import { Props } from '../Entity/types';
+import { Tile } from '../Tile/Tile';
+
+
+export default class Game {
+	protected ctx: CanvasRenderingContext2D;
+	public width = 1044;
+	public height = 778;
+	public startX = 20;
+	public startY = 10;
+	public squareSize = 64;
+	public squareInnerSize = this.squareSize - 2;
+
+	protected entities: Props[] = [];
+
+	constructor(protected canvas: HTMLCanvasElement) {
+		this.ctx = canvas.getContext('2d');
+	}
+
+	init(): void {
+		this.canvas.width = this.width;
+		this.canvas.height = this.height;
+        this.ctx.fillStyle = 'grey';
+        this.ctx.fillRect(this.startX, this.startY, 1044, 778);
+		this.initEntities();
+	}
+
+	initEntities(): void {
+		this.addEntity(new Tile(this.ctx, this.squareSize, this.squareSize, this.startX, this.startY, '#000000', '#ffffff', this.squareInnerSize, this.squareInnerSize, this.startX + 1, this.startY + 1));
+		for (let i=0; i<= 15; i++) {
```

Comment:
🧐
Кажется, что так выглядит чуть лучше
```suggestion
		for (let i = 0; i <= 15; i++) {
```

А вообще, тут есть много похожих свойств, которые ты передаёшь в тайлы, может, как-то их погруппировать?
И если в цикле выводятся разные тайлы, то было бы круто написать под них отдельные функции и тут просто вызвать их вместо написания циклов

Reply:
Здесь сгрупированно по вертикали и горизонтали